app/routes/_index.tsx

```diff
@@ -24,21 +25,35 @@ const sanityKlient = createClient({
   useCdn: true,
 });
 
-export const loader = async () => {
+export const loader: LoaderFunction = async () => {
   const data = await sanityKlient.fetch('*');
+
   return { data };
 };
 
 export default function Index() {
   const { data } = useLoaderData<typeof loader>();
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
```suggestion
  const [laster, settLaser] = useState(true);
```
Vi koder på norsk, så denne staten burde ha et norsk navn